<launch>
  <rosparam file="$(find objsearch_toplevel)/config/global_params.yaml" command="load" />
  
  <arg name="distance_threshold" default="0.04" />
  <param name="/planetest/RANSAC_distance_threshold" value="$(arg distance_threshold)" />
  <arg name="iterations" default="100" />
  <param name="/planetest/RANSAC_iterations" value="$(arg iterations)" />
  <arg name="planes_to_extract" default="5" />
  <param name="/planetest/planes_to_extract" value="$(arg planes_to_extract)" />
  <arg name="output" default="NULL"/>
  <param name="/planetest/output_dir" value="$(arg output)" />
  <arg name="cloud_file" />
  <param name="/planetest/cloud_file" value="$(arg cloud_file)" />

  <node name="planetest" pkg="preprocess" type="planetest" output="screen" />
</launch>
